Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,181 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
++----------------------------------+
+| Name                             |
++----------------------------------+
+| Aruba                            |
+| Anguilla                         |
+| Netherlands Antilles             |
+| Antigua and Barbuda              |
+| Bahamas                          |
+| Belize                           |
+| Bermuda                          |
+| Barbados                         |
+| Canada                           |
+| Costa Rica                       |
+| Cuba                             |
+| Cayman Islands                   |
+| Dominica                         |
+| Dominican Republic               |
+| Guadeloupe                       |
+| Grenada                          |
+| Greenland                        |
+| Guatemala                        |
+| Honduras                         |
+| Haiti                            |
+| Jamaica                          |
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+
+4. What's the top 10 countries by SurfaceArea ?
+select Name from country where country.SurfaceArea limit 0, 10;
++----------------------+
+| Name                 |
++----------------------+
+| Aruba                |
+| Afghanistan          |
+| Angola               |
+| Anguilla             |
+| Albania              |
+| Andorra              |
+| Netherlands Antilles |
+| United Arab Emirates |
+| Argentina            |
+| Armenia              |
++----------------------+
+10 rows in set (0.00 sec)
+
+5. What's the top 10 most populated cities?
+select Name, Population from city order by Population desc limit 10;
++---------------------+------------+
+| Name                | Population |
++---------------------+------------+
+| Mumbai (Bombay)     |   10500000 |
+| Seoul               |    9981619 |
+| SÃ£o Paulo          |    9968485 |
+| Shanghai            |    9696300 |
+| Jakarta             |    9604900 |
+| Karachi             |    9269265 |
+| Istanbul            |    8787958 |
+| Ciudad de MÃ©xico   |    8591309 |
+| Moscow              |    8389200 |
+| New York            |    8008278 |
++---------------------+------------+
+10 rows in set (0.12 sec)
+
+6. Select the top 3 countries by population that their names start with the letter ‘P’
+select Name, Population from city where Name like 'P%' order by Population desc limit 3;
```

Comment:
:+1:

Reply:
I actually read this again and realised you got it backwards - the question asks for 3 most populated COUNTRIES, not cities.